lib/models.py

```diff
@@ -225,7 +225,7 @@ def vote(self, dashd, signal, outcome):
             self.sync_network_vote(dashd, signal)
 
     def sync_network_vote(self, dashd, signal):
-        printdbg('\tsyncing network vote for object %s with signal %s' % (self.object_hash, signal.name))
+        printdbg('\tSyncing network vote for object %s with signal %s' % (self.object_hash, signal.name))
```

Comment:
Not technically a typo, and this _could_ affect if anyone has created a tool to look for specific text in the logs... but honestly probably not an issue. Will wait for more feedback to see what others think.